backend/src/Designer/Infrastructure/AnsattPorten/AnsattPortenConstants.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Altinn.Studio.Designer;
+
+public class AnsattPortenConstants
+{
+    public const string AnsattportenAuthenticationScheme = "AnsattportenAuthScheme";
+    public const string AnsattportenAuthorizationPolicy = "AnsattportenAuthenticated";
+    public const string AnsattpotenCookiesAuthenticationScheme = "AnsattportenCookies";
```

Comment:
Should cookies rather be singular, as in "cookie"? From what I can see, this is used for setting one Cookie in the browser?

```suggestion
    public const string AnsattpotenCookiesAuthenticationScheme = "AnsattportenCookie";
```

Reply:
Hmm. I was inspired by the default Cookies authentication scheme name which Is in plural `Cookies`, so I just added a prefix `Ansattporten`.

Reply:
ah, ok then we can keep it as is. From my point of view, it is one cookie with multiple properties. But let's keep it consistent with the default Cookies.